assets/js/common_utils.js

```diff
@@ -122,51 +122,43 @@ export const getTime = () => {
   return `${hour}:${min}:${sec}`;
 };
 
-// Get the latest dev version of ONNX Runtime Web dists
-const getLatestOrtWebDevVersion = async () => {
-  try {
-    const response = await fetch('https://cdn.jsdelivr.net/npm/onnxruntime-web/dist/');
-    const htmlString = await response.text();
-    const parser = new DOMParser();
-    const doc = parser.parseFromString(htmlString, 'text/html');
-    let selectElement = doc.querySelector('select.versions.select-css');
-    let options = Array.from(selectElement.querySelectorAll('option')).map(
-      (option) => option.value
-    );
-    let filteredOptions = options.filter(item => item.includes('-dev.'));
-    return filteredOptions[0].replace('onnxruntime-web@', '');
-  } catch (error) {
-    console.error('Error:', error.message);
-  }
+const KNOWN_COMPATIBLE_ORT_VERSION = {
+  'stable-diffusion-1.5': { 'dev': '1.19.0-dev.20240804-ee2fe87e2d', 'stable': '1.20.0', 'test': 'test' },
+  'sd-turbo': { 'dev': '1.19.0-dev.20240804-ee2fe87e2d', 'stable': '1.20.0', 'test': 'test' },
+  'segment-anything': { 'dev': '1.19.0-dev.20240804-ee2fe87e2d', 'stable': '1.20.0', 'test': 'test' },
+  'whisper-base': { 'dev': '1.19.0-dev.20240804-ee2fe87e2d', 'stable': '1.20.0', 'test': 'test' },
+  'image-classification': { 'dev': '1.19.0-dev.20240804-ee2fe87e2d', 'stable': '1.20.0', 'test': 'test' },
```

Comment:
Is this unreferenced?

Reply:
Never mind. Image classification uses transformers.js and a static version of ORT.